graphql/resolvers/addComment.ts

```diff
@@ -11,6 +11,137 @@ import {
   PRIMARY_COLOR_HEX,
   PROFILE_URL
 } from '../../constants'
+import { Challenge, Comment, Lesson, Submission } from '@prisma/client'
+import { MessageResponse } from '../../@types/discordBot'
+
+type CommentWithAuthorAndSubmission = Comment & {
+  author: {
+    id: number
+    username: string
+    discordId: string | null
+    name: string
+  }
+  submission: Submission & {
+    user: { id: number; username: string; discordId: string | null }
+    lesson: Lesson
+    challenge: Challenge
+  }
+}
+
+type SendNotificationsParams = {
+  line: InputMaybe<number> | undefined
+  fileName: InputMaybe<string> | undefined
+  submissionId: number
+  comment: CommentWithAuthorAndSubmission
+}
+
+const sendNotifications = async ({
+  line,
+  submissionId,
+  fileName,
+  comment
+}: SendNotificationsParams) => {
+  const restOfComments = await prisma.comment.findMany({
```

Comment:
You need to add the distinct option here on `authorId` to make prisma select an author only once https://www.prisma.io/docs/concepts/components/prisma-client/aggregation-grouping-summarizing#select-distinct

Reply:
Strange. I'm certain I added this option and staged the changes.

Reply:
Anyway, I'll add it now.